src/analysis/locally_convex/bounded.lean

```diff
@@ -0,0 +1,117 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+import analysis.locally_convex.basic
+import topology.bornology.basic
+
+/-!
+# Natural Boundedness
+
+This file defines natural or von Neumann bounded sets and proves elementary properties.
+
+## Main declarations
+
+* `is_vonN_bounded`: A set `s` is von Neumann-bounded if every neighborhood of zero absorbs `s`.
```

Comment:
Please make this and other names in these docstrings be fully qualified so that they turn into links in the generated documentation.
```suggestion
* `bornology.is_vonN_bounded`: A set `s` is von Neumann-bounded if every neighborhood of zero absorbs `s`.
```